.github/workflows/release.yml

```diff
@@ -89,9 +89,16 @@ jobs:
     needs: wait-for-package-release
     uses: ./.github/workflows/publish_docker_image.yml
     secrets: inherit
-  publish-helm-chart:
+  publish-codespace-image:
```

Comment:
I'm not sure why this needs to happen before the helm chart necessarily? As this doesn't actually block the OSS release, I would have this at the end of the GH action somewhere, where it failing doesn't actually cause some crucial release actions to not start.